low-level/solution/lr2/server/health.py

```diff
@@ -161,11 +161,11 @@ def set_health_data(health_data: dict, status, description=None,
                         recommendation=None, specifics=None):
         """Sets health attributes for a component."""
         good_state = (status == "OK")
-        if not description:
+        if description in const.RESOURCE_MAP["undesired_values"]:
             description = "%s %s in good health." % (
                 health_data.get("uid"),
                 'is' if good_state else 'is not')
-        if not recommendation:
+        if recommendation in const.RESOURCE_MAP["undesired_values"]:
             recommendation = 'NA' if good_state\
```

Comment:
if health is good, recommendation needs to be None, not NA

Reply:
By default, "NA" is coming from enclosure response for health 'OK'.
Please confirm once. Currently, we used to keep "NA" for any attribute if None.

Reply:
Hi I think, we should use `None` and `NotAvailable` in it's full form in corresponding situations.
like the recommendation can be `None` and some value or info that we failed to retrieve can be `NotAvailable`.

Reply:
Corrected for recommendation.

Reply:
Done.